.zenodo.json

```diff
@@ -0,0 +1,120 @@
+{
+    "creators": [
+      {
+        "affiliation": "HZDR, TU Dresden",
+        "name": "Huebl, Axel",
+        "orcid": "0000-0003-1943-7141"
+      },
+      {
+        "affiliation": "HZDR",
+        "name": "Widera, René",
+        "orcid": "0000-0003-1642-0459"
+      },
+      {
+        "affiliation": "LogMeIn, Inc.",
+        "name": "Worpitz, Benjamin"
+      },
+      {
+        "affiliation": "HZDR, TU Dresden",
+        "name": "Pausch, Richard",
+        "orcid": "0000-0001-7990-9564"
+      },
+      {
+        "affiliation": "HZDR, TU Dresden",
+        "name": "Burau, Heiko"
+      },
+      {
+        "affiliation": "HZDR, TU Dresden",
+        "name": "Garten, Marco",
+        "orcid": "0000-0001-6994-2475"
+      },
+      {
+        "affiliation": "HZDR",
+        "name": "Starke, Sebastian"
+      },
+      {
+        "affiliation": "HZDR, TU Dresden",
```

Comment:
should HZDR be still there?

Reply:
yes, contributed to PIConGPU while under HZDR employment during his MA/Dipl thesis at TU DD.

Reply:
solved - that's fine with me

Reply:
ping @Flamefire : whatever you prefer (see comments above)

Reply:
I don't mind. Not sure what the correct interpretion of this is so I'd go with "at the time contributed"